### ISSUE TITLE
Not working on libinput 1.19.x

### ISSUE THREAD

**Issue by iberianpig** [iberianpig/fusuma-plugin-tap#14]
ref: https://github.com/iberianpig/fusuma-plugin-tap/pull/8#issuecomment-945948988

**Comment by mohamed-ibrahem** [comment]
Are you working on it?

**Comment by iberianpig** [comment]
I have the will to work on it, but I don't have the time to do it now.
We need to add some tests and improve the parser plugin to support libinput1.19.x.

**Comment by mohamed-ibrahem** [comment]
Yes I know, In all honesty, I hope that work will be done to resolve this matter because it is a wonderful project.
